res/rules-engine/configuration.toml

```diff
@@ -74,6 +74,22 @@ PublishTopic="rules-events"
     Host = "localhost"
     Port = 6379
     Protocol = "redis"
+    [Trigger.EdgexMessageBus.Optional]
+    authmode = 'usernamepassword'  # required for redis messagebus (secure or insecure).
+    secretname = 'redisdb'
+    # Default MQTT Specific options that need to be here to enable evnironment variable overrides of them
+    # Client Identifiers
+    ClientId ="core-data"
+    # Connection information
+    Qos          =  "0" # Quality of Sevice values are 0 (At most once), 1 (At least once) or 2 (Exactly once)
+    KeepAlive    =  "10" # Seconds (must be 2 or greater)
```

Comment:
should we consider to use time duration like "10s" instead of relying on comment?

Reply:
No, this is specific to MQTT implementation of go-mod-messaging which simply passes it to the mqtt go module